pytorch_lightning/core/lightning.py

```diff
@@ -1493,6 +1493,42 @@ def configure_gradient_clipping(self, optimizer, optimizer_idx, gradient_clip_va
             optimizer, gradient_clip_val=gradient_clip_val, gradient_clip_algorithm=gradient_clip_algorithm
         )
 
+    def lr_scheduler_step(
+        self,
+        scheduler: Any,
+        optimizer_idx: Optional[int] = None,
+        metrics: Optional[Union[float, torch.Tensor]] = None,
+    ) -> None:
+        r"""
+        Override this method to adjust the default way the
+        :class:`~pytorch_lightning.trainer.trainer.Trainer` calls each scheduler.
+        By default, Lightning calls ``step()`` and as shown in the example
+        for each scheduler based on its ``interval``.
+
+        Args:
+            scheduler: Learning rate scheduler.
+            optimizer_idx: Index of the optimizer associated with scheduler.
+            monitor_val: Value of the metric used for schedulers like ``ReduceLROnPlateau``.
+
+        Examples::
+
+            # DEFAULT
+            def lr_scheduler_step(self, scheduler, optimizer_idx, metrics):
+                if monitor_val is None:
+                    scheduler.step()
+                else:
+                    scheduler.step(monitor_val)
+
+            # Alternative way to update schedulers if it requires an epoch value
+            def lr_scheduler_step(self, scheduler, optimizer_idx, metrics):
+                scheduler.step(epoch=self.current_epoch)
+
+        """
+        if metrics is None:
+            scheduler.step()
+        else:
+            scheduler.step(metrics=metrics)
```

Comment:
This doesn't make sense considering that most schedulers won't take this argument. Previously, it was passed positionally, and wasn't plural

Reply:
updaed!